### PR TITLE
Set up CI/CD

### LINKED ISSUE #2: Set up continuous integration
- [x] Set up TravisCI and connect to repo 
- [x] Build successfully deploys

**Resources**
- [Building a Python Project](https://docs.travis-ci.com/user/languages/python/)

### PR DESCRIPTION
**Pull request by freeheeling**
## What functionality does this accomplish?
addresses #2 
​
**Description:**
Configure travis yml file

## What did you struggle on to complete?
Running `$ travis encrypt $(heroku auth:token) --add deploy.api_key --pro`​ did not initially return an encrypted key that could be added to `.travis.yml`. After uninstalling the travis gem, I ended up running `brew install travis`, which resolved the issue.
Overall configuration was challenging. Credit is due to Linda for devising a solution.

### Test Coverage Percentage: 100%
- [X] No Tests have been changed
- [x] New Tests have been implemented
- [ ] Some Tests have been changed
- [ ] All of the Tests have been changed(Please describe what in the world happened):
​
## Checklist:
- [x] My code has no unused/commented out code
- [x] I have reviewed my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have fully tested my code
- [ ] I have partially tested my code (please explain why):
​
## Helpful Resources:
* Resource link AND small description of info gathered/learned
https://docs.travis-ci.com/user/deployment/heroku/​

## Review Requests(optional):
A close review to verify I haven't missed anything would be appreciated.

## Please include a gif of how you feel about this branch:
Alt Text

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+language: python
+python:
+  - 3.8
+before_install:
+  - pip install -U pip
+install:
+  - pip install -r requirements.txt
+script:
+  - pytest
```
Comment (Jonpatt92) [inline .travis.yml:9]: Nice scripting configuration!

**Review by Jonpatt92 — APPROVED** [review]
Very nice job setting up CI/CD

File: app_test.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import requests
+def test_get_locations_for_us_90210_check_city_equals_beverly_hills():
```
Comment (linda-le1) [inline app_test.py:2]: Looks like pytest returns a fail if run on TravisCI with no tests (https://github.com/pytest-dev/pytest/issues/2239) so thank you for putting in a passing test.

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+venv
```
Comment (linda-le1) [inline .gitignore:1]: I looked at a few open source python projects and the virtual environments have been in the gitignore, so I agree that this is the way to go. Nice catch!

**Review by linda-le1 — APPROVED** [review]
Nice work on the configuration set up! Thank you.

**Pull request merged**